constraint/pkg/client/drivers/local/local.go

```diff
@@ -75,22 +85,82 @@ func New(args ...Arg) drivers.Driver {
 	for _, arg := range args {
 		arg(d)
 	}
+
+	// adding externaldata builtin otherwise capabilities get overridden
+	// if a capability, like http.send, is disabled
+	if d.providerCache != nil {
+		d.capabilities.Builtins = append(d.capabilities.Builtins, &ast.Builtin{
+			Name: "externaldata",
+			Decl: opatypes.NewFunction(opatypes.Args(opatypes.S, opatypes.S), opatypes.A),
```

Comment:
It seems like requiring a string may be a bit limiting... what if a user wants to pass a more complex object?

Also... is there any way for a user to batch requests?

Reply:
I think we said we'll start with a string with this to keep things simple and consistent with mutation.

Reply:
I'm not sure that would be consistent with mutation as-written?

Mutation is allowing the user to pass the value of the field being mutated. That could be a:

- string
- struct
- list

basically any type.

If we want to try to keep things simple, that works for me, but whatever limitations we impose should be enforceable, forward-compatible, and consistent between mutation and validation.

WRT batching, I remember discussing accepting a list of requests for the argument, which would govern the batching behavior?

Reply:
added primitive type check for validation and rego should be able to batch requests using a list.